Tasks/AzureCLI/azureclitask.ts

```diff
@@ -44,13 +54,18 @@ export class azureclitask {
             tl.mkdirP(cwd);
             tl.cd(cwd);
 
+            if (scriptType === "bash") {
+                tool.pathArg(scriptPath);
+            }
+            else {
+                tool = tl.createToolRunner(tl.which(scriptPath, true));
+            }
+
             var connectedServiceNameSelector = tl.getInput("connectedServiceNameSelector", true);
             this.loginAzure(connectedServiceNameSelector);
 
-            bash.pathArg(scriptPath);
-            bash.argString(args); // additional args should always call argString.  argString() parses quoted arg strings
-
-            await bash.exec({failOnStdErr: failOnStdErr});
+            tool.argString(args); // additional args should always call argString.  argString() parses quoted arg strings
```

Comment:
Great.  Don't forget to add a package.json like the ShellScript task did.  That drives our build process to pull in that exact version of task lib.  You might have to edit the .d.ts in the task repo since it's a union of the interfaces in the versions of task lib.  It's a diamond dependency problem which we are going to address very soon.  let me know if you have an issue.